source/PlayerInfo.cpp

```diff
@@ -3236,7 +3235,25 @@ void PlayerInfo::RegisterDerivedConditions()
 	conditions["day"].ProvideNamed([this](const ConditionEntry &ce) { return date.Day(); });
 	conditions["month"].ProvideNamed([this](const ConditionEntry &ce) { return date.Month(); });
 	conditions["year"].ProvideNamed([this](const ConditionEntry &ce) { return date.Year(); });
-	conditions["weekday"].ProvideNamed([this](const ConditionEntry &ce) { return date.WeekdayNumber(); });
+	conditions["weekday: "].ProvidePrefixed([this](const ConditionEntry &ce) -> int64_t {
+		string day = ce.NameWithoutPrefix();
+		int number = date.WeekdayNumber();
```

Comment:
If anyone complains about this function still outputting 1 to 7, Sunday to Saturday, I'm changing it to just read the raw output of WeekdayNumberOffset so that Saturday outputs as 0 and Friday outputs as 6.

Reply:
Actually, that's what the Weekday function that outputs the weekday abbreviation string does, so I'll just change it anyway.